machine.go

```diff
@@ -523,6 +529,15 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 				i, i, img.label))
 	}
 
+	if m.qemuopts != nil {
+		keyval := strings.Fields(m.qemuopts[0])
+		for _, v := range keyval {
+			qemuargs = append(qemuargs, v)
+		}
+	} else {
+		qemuargs = append(qemuargs, "-nographic")
```

Comment:
isn't -nographic already added above?

Reply:
I moved `-nographic` from above to this else part. But didn't make that change while creating patch. Will make the change and update the patch. I was under the assumption` -nographic` and `-display` cannot be passed simultaneously. But qemu works fine even when both args are passed. So should I always keep `nographic` regardless of `-display` used or not?

Reply:
See also #29 which switches from `-nographic` to `-display none`, and explicitly sets up the ttys separately (`-nographic` does multiple things, which can be confusing). It might be better to have an explicit `--display` option (mapped to qemu's `-display`, and defaulting to `none`) instead of passing it in via an "arbitrary qemu options" interface.

I wonder whether, longer-term, passing `-nodefaults` and setting everything up explicitly would be a better approach.

Reply:
We initially discussed about using explicit qemu specific `-qemudisplay` option, later decided - we should accept arbitrary values since it will allow us to include more  options in future  without changing fake-machine parts.

Reply:
Okay, will remove `-nographic` and re-submit patch